pages/authorDashboard/PKPAuthorDashboardHandler.inc.php

```diff
@@ -157,23 +159,34 @@ function setupTemplate($request) {
 			if ($fileStage && is_a($lastReviewRound, 'ReviewRound')) {
 				$editorDecisions = DAORegistry::getDAO('EditDecisionDAO')->getEditorDecisions($submission->getId(), $submission->getData('stageId'), $lastReviewRound->getId());
 				if (!empty($editorDecisions) && array_last($editorDecisions)['decision'] == SUBMISSION_EDITOR_DECISION_PENDING_REVISIONS) {
```

Comment:
I know it's not your code, but we need to add the other possible revisions decision to this list, `SUBMISSION_EDITOR_DECISION_RESUBMIT`. So this line would be:

```
$lastDecision = array_last($editorDecisions)['decision'];
$revisionDecisions = [SUBMISSION_EDITOR_DECISION_PENDING_REVISIONS, SUBMISSION_EDITOR_DECISION_RESUBMIT];
if (!empty($editorDecisions) && in_array($lastDecision, $revisionDecisions)) {
```

Reply:
check